File: go/hackerrank/warm-up/plusMinus.go
// https://www.hackerrank.com/challenges/plus-minus/problem

func plusMinus(arr []int32) {
    positiveCnt := 0
    negativeCnt := 0
    zeroCnt := 0
    length := len(arr)

    for _, num := range(arr) {
        if (num > 0){
            positiveCnt += 1
        }
        if (num < 0){
            negativeCnt += 1
        }
        if (num ==0){
            zeroCnt += 1
        }
    }
    fmt.Printf("%f\n", float32(positiveCnt)/float32(length))
    fmt.Printf("%f\n", float32(negativeCnt)/float32(length))
    fmt.Printf("%f\n", float32(zeroCnt)/float32(length))
}

File: python/hackerrank/archive/20.08/plus_minus.py
# https://www.hackerrank.com/challenges/plus-minus/problem

def plusMinus(arr):
    positive_cnt = 0
    negative_cnt = 0
    zero_cnt = 0
    length = len(arr)

    for num in arr:
        if num > 0:
            positive_cnt += 1
            continue
        if num < 0:
            negative_cnt += 1
            continue
        zero_cnt += 1

    print("{:f}".format(positive_cnt/length))
    print("{:f}".format(negative_cnt/length))
    print("{:f}".format(zero_cnt/length))
